django_x509/base/admin.py

```diff
@@ -150,7 +150,7 @@ class AbstractCertAdmin(BaseAdmin):
     actions = ['revoke_action']
 
     class Media:
-        js = ('django-x509/js/x509-admin.js',)
+        js = ('admin/js/jquery.init.js', 'django-x509/js/x509-admin.js',)
```

Comment:
Can you move it to the next line? 
(Curtesy: pep8 -- read about it if you haven't already! :smile:)